packages/core/demo/demo-data/bar.ts

```diff
@@ -92,6 +108,22 @@ export const simpleBarOptions = {
 	}
 };
 
+// Horizontal Simple
+export const simpleHorizontalBarData = simpleBarData;
+
+export const simpleHorizontalBarOptions = {
+	title: "Simple horizontal bar (discrete)",
+	axes: {
+		left: {
+			primary: true,
+			scaleType: "labels"
+		},
+		bottom: {
+			secondary: true,
+		}
+	},
+	orientation: "horizontal"
```

Comment:
From what I can see we should be able to remove this after updating `graphs/bar`